### DISCUSSION TITLE
ideal number of emqx nodes

### DISCUSSION THREAD

**Discussion by alexa22013** [emqx/emqx#9656] [General]
is there any doc to know the ideal number of core and replicant nodes in emqx v5 based on the requirements?

Thanks in advance...

**Comment by thalesmg** [comment]
Hi @alexa22013 !

We have some general suggestions described here: https://www.emqx.io/docs/en/v5.0/deploy/cluster/db.html#configuration

However, for an _ideal_ configuration, you'll need to benchmark the cluster to your particular use cases and tune accordingly.
